default/customizations/common_user_customizations.txt

```diff
@@ -109,6 +109,10 @@ SITREP_MONSTER_FLEET_ARRIVED_AT_DESTINATION_LABEL
 SITREP_MONSTER_SHIP_ARRIVED_AT_DESTINATION_LABEL
 SITREP_FLEET_ARRIVED_AT_DESTINATION_LABEL
 SITREP_PLANET_BOMBARDED_AT_SYSTEM_LABEL
-SITREP_UNOWNED_PLANET_BOMBARDED_AT_SYSTEM_LABEL'''
+SITREP_UNOWNED_PLANET_BOMBARDED_AT_SYSTEM_LABEL
+SITREP_SHIP_REPAIR_DOCK_NOPOP
+SITREP_SHIP_REPAIR_DOCK_NONE
+SITREP_SHIP_REPAIR_DOCK_PARTIAL
+SITREP_SHIP_REPAIR_DOCK_COMPLETE'''
```

Comment:
This will have the side effect that drydock repair sitreps appear above all the sitreps that have no entry in this list (thus getting moved up), but that shouldn't be an issue.